tests/components/fujitsu_fglair/test_sensor.py

```diff
@@ -31,3 +31,20 @@ async def test_entities(
     assert await integration_setup()
 
     await snapshot_platform(hass, entity_registry, snapshot, mock_config_entry.entry_id)
+
+
+async def test_no_outside_temperature(
+    hass: HomeAssistant,
+    entity_registry: er.EntityRegistry,
+    snapshot: SnapshotAssertion,
+    mock_ayla_api: AsyncMock,
+    mock_config_entry: MockConfigEntry,
+    integration_setup: Callable[[], Awaitable[bool]],
+) -> None:
+    """Test that coordinator returns the data we expect after the first refresh."""
+    for d in mock_ayla_api.async_get_devices.return_value:
+        d.outdoor_temperature = None
```

Comment:
```suggestion
    for device in mock_ayla_api.async_get_devices.return_value:
        device.outdoor_temperature = None
```

Reply:
I was wondering if we could have 1 device with `outdoor_temperature` and another(s) without, and check that `len()` is 1 instead?
It makes the test a bit more robust so that we check that it is actually only filtering that (instead of the integration not getting properly initialized, for example)